sdk/tables/aztables/recordings/TestServiceClient_Cosmos/TestCreateTable.yaml

```diff
@@ -0,0 +1,75 @@
+---
+version: 1
+interactions:
+- request:
+    body: '{"TableName":"gotablecwmvbx3xi23m7"}'
+    form: {}
+    headers:
+      Accept:
+      - application/json;odata=minimalmetadata
+      Authorization:
+      - sanitized
+      Content-Length:
+      - "36"
+      Content-Type:
+      - application/json
+      Dataserviceversion:
+      - "3.0"
+      User-Agent:
+      - azsdk-go-tables/<version> azsdk-go-tables/<version> azcore/v0.13.4 (go1.16.2;
+        Windows_NT)
+      X-Ms-Date:
+      - Sun, 04 Apr 2021 21:54:53 GMT
+      X-Ms-Version:
+      - "2019-02-02"
+    url: https://chrisstablesprim.table.cosmos.azure.com/Tables
+    method: POST
+  response:
+    body: '{"TableName":"gotablecwmvbx3xi23m7","odata.metadata":"https://chrisstablesprim.table.cosmos.azure.com/$metadata#Tables/@Element"}'
```

Comment:
How do we sanitize the body? If we want to clean the url in odata.metadata?

Reply:
Good catch - I need to add a `santizeBody` func to `recording_sanitizer`